README.md

```diff
@@ -438,69 +438,13 @@ The design system also meets the [WCAG 2.0 AA accessibility guidelines](https://
 
 ## Pattern library
 
-We're using [Pattern lab](https://patternlab.io/) to generate an interactive component library for the design system. You can run it locally after `npm install` with:
+We're using [StorybookJS](https://storybook.js.org/) to generate an interactive component library for the design system. You can run it locally after `npm install` with:
 
 ```sh
 npm start
 ```
 
-Then, visit [http://localhost:3000/](http://localhost:3000/) to see the design system in action. It will watch for changes in twig, scss, and JS and refresh.
-
-
-### Template compatibility
```

Comment:
We should remove the "Template compatibility" link in the [Table of Contents](https://github.com/uswds/uswds/blob/23b1c3733027ed380d9575584485052c56c2ba3e/README.md?plain=1#L33)

Reply:
Good point! Removed in 009d8fcebbbf4f07a7a6b640786b096959070d4c.